EnglishFilter/sections/antiadblock.txt

```diff
@@ -411,6 +411,8 @@ mstream.cloud#%#AG_setConstant('cRads', 'true');
 @@||direct-link.net^$generichide
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_safari, ext_ublock)
 @@.gif#$domain=direct-link.net
+! https://github.com/AdguardTeam/AdguardFilters/issues/35399
+@@||dom2.ru^$generichide
```

Comment:
Move to Russian

Reply:
I noticed this message when Base filter is on. That's why i added exclusion in Base, not Russian.

Reply:
Still Russian site. Else a lot of Russian sites would be in Base filter.